### PR TITLE
VBLOCKS-1801 Example of starting a call from iOS Phone app history

### PR DESCRIPTION
**Pull request by bobiechen-twilio**
## Submission Checklist

 - [ ] The `CHANGELOG.md` reflects any **feature**, **bug fixes**, or **known issues** made in the source code
 - [ ] New unit tests and integration tests have beed added
 - [ ] Code coverage is more or equal to the previous coverage
 - [ ] A visual inspection of the `Files changed` tab was made prior to submitting the pull request ensuring the style guide was followed
 - [x] [CI pipeline](https://app.circleci.com/pipelines/github/twilio/twilio-voice-react-native-app) is green
 - [ ] Included screenshot if necessary

## Description

Example of starting a call from iOS Phone app history

## Breakdown

- Add example code to the reference app
- Update README and instructions

## Additional Notes

This will require modifications to the local copy of the twilio-voice-react-native SDK source code. See the instructions in the markdown file.

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

**Review by charliesantos — COMMENTED** [review]
Instead of a tutorial, I think we need to implement this in the reference app, which is the purpose of this repo. It's a bit weird adding example code in docs in a reference app repo. Thoughts?

**Comment by bobiechen-twilio** [comment]
The actually implementation is also in the PR, just that it needs some extra explanation and instructions to modify the iOS app project settings and the ObjC code of the RN SDK.

File: app/README.md [hunk 1]
```diff
@@ -94,3 +94,7 @@ See the Redux-Toolkit documentation [here](https://redux-toolkit.js.org/introduc
 React Navigation is used to provide "native"-feeling navigation for the application.
 
 See the React Navigation documentation [here](https://reactnavigation.org/docs/getting-started/).
+
+## Examples
+
+- [Starting a call from iOS Phone app history](docs/ios-call-from-recents.md)
```
Comment (charliesantos) [inline app/README.md:100]: I think this should go between **Incoming call notifications** and **Setting up Auth0** above this readme. Something like
```
## Call history

Please check this [page](docs/ios-call-from-recents.md) for details on how to start a call from the iOS Phone app history.
```

**Comment by charliesantos** [comment]
> The actually implementation is also in the PR, just that it needs some extra explanation and instructions to modify the iOS app project settings and the ObjC code of the RN SDK.

Sorry, I'm not too familiar with iOS 🤦 
I left one comment. Please take a look.

**Review by charliesantos — APPROVED** [review]

**Pull request merged**